docs/Installing_and_RunningTheHPCCPlatform/Inst-Mods/ssl-esp.xml

```diff
@@ -1,383 +1,600 @@
+            </listitem>
+
+            <listitem>
+              <para>Press the <emphasis role="bold">Next</emphasis>
+              button.</para>
+
+              <para>The Configuration Manager Advanced View interface
+              displays.</para>
+            </listitem>
+
+            <listitem>
+              <para>Check the <emphasis role="bold">Write Access
+              </emphasis>box at the top of the page.</para>
+            </listitem>
+          </orderedlist>
+        </sect3>
+
+        <sect3 id="Configure_Roxie_forHTTPS" role="brk">
+          <title><emphasis role="bold">Configure Roxie SSL</emphasis></title>
+
+          <orderedlist numeration="arabic">
+            <listitem>
+              <para>Select your Roxie Cluster in the Navigator panel on the
+              left side.</para>
+            </listitem>
+
+            <listitem>
+              <para>Select the <emphasis role="bold">Ports</emphasis>
+              tab.</para>
+            </listitem>
+
+            <listitem>
+              <para>Right-click on the list of ports, then select <emphasis
+              role="bold">Add</emphasis>.<figure>
+                  <title>Select Port Tab</title>
+
+                  <mediaobject>
+                    <imageobject>
+                      <imagedata fileref="../../images/CM-RoxSSL01.jpg" />
+                    </imageobject>
+                  </mediaobject>
+                </figure></para>
+            </listitem>
+
+            <listitem>
+              <para>The default port number is 9876. Change the default port
+              number, for example, to 19876.</para>
+            </listitem>
+
+            <listitem>
+              <para>Change the protocol from <emphasis>Native</emphasis> to
+              <emphasis>SSL</emphasis> from the drop menu (image).</para>
+            </listitem>
+
+            <listitem>
+              <para>In the <emphasis role="bold">passphrase</emphasis> entry
+              control, enter the passphrase you used earlier when you created
+              the private key. Leave this field empty if you did not use a
+              passphrase.</para>
+            </listitem>
+
+            <listitem>
+              <para>When prompted, provide the passphrase again.</para>
+            </listitem>
+
+            <listitem>
+              <para>Enter the certfilename.</para>
+            </listitem>
+
+            <listitem>
+              <para>Enter the keyfilename.</para>
+            </listitem>
+
+            <listitem>
+              <para>Click the disk icon to save.</para>
+            </listitem>
+          </orderedlist>
+
+          <para>The default lookup location for the cert and key files is in
+          <emphasis>/var/lib/HPCCSystems/myroxie</emphasis>. Unless you
+          specify differently and provide the full path this is the location
+          where you would copy the cert and key files for each Roxie
+          node.</para>
```

Comment:
The default location for the certificate and key files is <emphasis>/var/lib/HPCCSystems/myroxie</emphasis>. You can specify a full path if you want these files in a different location. The certificate and key must <emphasis role="bold">must</emphasis> be available to every Roxie node.